engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -534,26 +543,28 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
         case _ => // The WMA is waiting for the WorkflowActorWorkComplete message. No extra information needed here.
       }
 
-      // Copy/Delete workflow logs
```

Comment:
This section is just a copy/paste of this code to move the brute force workflow options up in scope (outside `if (WorkflowLogger.isEnabled)`.